<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>garage</class>
 <widget class="QMainWindow" name="garage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(&quot;c:/users/game/Desktop/smart - Copie (2)/guest_backgrounds/garage.png&quot;)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>533</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi Cond</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #2c3e50, stop: 0.4 #2c3e50,
                                stop: 0.5 #2c3e50, stop: 1.0 #2c3e50);
    border: 2px solid #7f8c8d;
    border-bottom-color: #7f8c8d; /* same as the pane color */
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    min-width: 255px;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #2980b9, stop: 0.4 #2980b9,
                                stop: 0.5 #2980b9, stop: 1.0 #ecf0f1);
}

QTabBar::tab:selected {
    border-color: #7f8c8d;
    border-bottom-color: #7f8c8d; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 5px; /* make non-selected tabs look smaller */
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="view">
     <attribute name="title">
      <string>VIEW</string>
     </attribute>
     <widget class="QTableWidget" name="viewgarage">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="open">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OPEN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="close">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
     <widget class="QLabel" name="ViewBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>533</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>guest_backgrounds/guest2-01.png</pixmap>
      </property>
     </widget>
     <zorder>ViewBackground</zorder>
     <zorder>viewgarage</zorder>
     <zorder>open</zorder>
     <zorder>close</zorder>
    </widget>
    <widget class="QWidget" name="add">
     <attribute name="title">
      <string>ADD</string>
     </attribute>
     <widget class="QPushButton" name="addgarage">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLabel" name="garagelogo2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(&quot;C:/Users/game/Desktop/smart/gfx/Finalcons/garage.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="addid">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: lightgrey;
    selection-background-color: darkgray;
}</string>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="AddBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>533</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>guest_backgrounds/guest2-01.png</pixmap>
      </property>
     </widget>
     <zorder>AddBackground</zorder>
     <zorder>addgarage</zorder>
     <zorder>garagelogo2</zorder>
     <zorder>addid</zorder>
    </widget>
    <widget class="QWidget" name="remove">
     <attribute name="title">
      <string>REMOVE</string>
     </attribute>
     <widget class="QLabel" name="garagelogo3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(&quot;C:/Users/game/Desktop/smart/gfx/Finalcons/garage.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="viewgarage2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>260</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="autoScrollMargin">
       <number>0</number>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="removegarage">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Snap ITC</family>
       </font>
      </property>
      <property name="text">
       <string>REMOVE</string>
      </property>
     </widget>
     <widget class="QLabel" name="RemoveBackground">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>533</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>guest_backgrounds/guest2-01.png</pixmap>
      </property>
     </widget>
     <zorder>RemoveBackground</zorder>
     <zorder>garagelogo3</zorder>
     <zorder>viewgarage2</zorder>
     <zorder>removegarage</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(&quot;C:/Users/game/Desktop/smart - Copie (2)/gfx/Finalcons/padlock.png&quot;)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
